arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -83,22 +100,45 @@
             "$ref": "#/parameters/apiVersionParameter"
           },
           {
-            "name": "resourceGroupName",
-            "description": "Name of the resource group within the Azure subscription.",
-            "in": "path",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/resourceGroupNameParameter"
           }
         ],        
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success. Returns a list of all of the Media Services accounts in a resource group.",
             "schema": {
               "$ref": "#/definitions/MediaServiceCollection"
+            },
+            "examples": {
+              "application/json": {
+                "value": [
+                  {
+                    "name": "contosomedia",
+                    "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Media/mediaservices/contosomedia",
+                    "type": "Microsoft.Media/mediaservices",
+                    "location": "West US",
+                    "tags":null,
+                    "properties":{  
+                      "apiEndpoints":[  
+                        {  
+                          "endpoint": "https://wamsbayclus001rest-hs.cloudapp.net/api/",
+                          "majorVersion": "2"
+                        }
+                      ],
+                      "storageAccounts":[  
+                        {  
+                          "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Storage/storageAccounts/contosomedia",
+                          "isPrimary": true
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }
             }
           },
           "default": {
-            "description": "Detailed error information.",
+            "description": "List Media Services failed.",

```

Comment:
Would this be better as "List Media Services failures.", or "List Media Services that failed." ?